projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -165,7 +164,6 @@ if (environment.requestedDeliveryDate) {
     UserModule,
     UserOccModule,
     // User UI
-    AddressBookModule,
```

Comment:
This import right here makes me wonder if this was intended for an AddressBook to be separate.